src/lib/cloud-manager-hoc.jsx

```diff
@@ -126,15 +126,23 @@ const cloudManagerHOC = function (WrappedComponent) {
     CloudManager.propTypes = {
         canSave: PropTypes.bool.isRequired,
```

Comment:
Also confused why this is a prop for CloudManager, even though you haven't touched that.

Reply:
I don't know what you mean about having touched that. For why `canSave` is a prop here, see my comment above, at https://github.com/LLK/scratch-gui/pull/5198#discussion_r362549403

Reply:
By "you haven't touched that", I was just acknowledging that this line isn't necessarily in scope for this PR since it's unchanged.